### PR TITLE
CLDR-8038 Revise index.md, translation.md, getting-started.md, guide.md

### PR DESCRIPTION
**Pull request by btangmu**
-Fix some broken links (like Updates), some found manually, others found using lychee

-Made all links (except images) relative to docs/site

-Do not say Survey tool is currently open for submission

-Concerning some emoji news, change These will be released... to These were released...

-Make the text "and many others, including:" part of the following list item, to avoid irregular formatting

-Uniformly call the Info Panel the Info Panel rather than the Information Panel, Info Pane, etc.

-Change http: to https:

-Remove News item dated 2022-11-28

CLDR-8038

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

File: docs/site/index.md [hunk 1]
```diff
@@ -6,24 +6,24 @@ title: Unicode CLDR Project
 
 ## News
 
-- **2024-09-26 [CLDR 46 Beta](https://blog.unicode.org/2024/09/unicode-cldr-46-beta-available-for.html) available for testing and specification review**
-- **2024-09-05 [CLDR 46 Alpha](https://blog.unicode.org/2024/09/unicode-cldr-v46-alpha-available-for.html)  available for testing**
-- **2024-04-17 [CLDR 45](/index/downloads/cldr-45) released**
-- **2023-12-13 [CLDR 44.1](/index/downloads/cldr-44#h.nvqx283jwsx) released (an update to CLDR v44)**
-- **2023-10-31 [CLDR 44](/index/downloads/cldr-44) released**
+- **2024-09-26 [CLDR 46 Beta] (https://blog.unicode.org/2024/09/unicode-cldr-46-beta-available-for.html) available for testing and specification review**
```
Comment (macchiati) [inline docs/site/index.md:9]: You can't put a space between the ] and ( in the link.

**Review by macchiati — CHANGES_REQUESTED** [review]
It's too hard to review until that is fixed.

File: docs/site/index.md (hunk 1 above)
Comment (srl295) [reply docs/site/index.md:9]: correct

**Review by srl295 — COMMENTED** [review]
looks like some good improvements

File: docs/site/index.md (hunk 1 above)
Comment (btangmu) [reply docs/site/index.md:9]: Live and learn! I'll fix that shortly...

Comment (btangmu) [reply docs/site/index.md]: Fixed by 2nd commit

**Comment by btangmu** [comment]
my latest commit fixes lots of links found by lychee running locally with docker

File: docs/site/translation.md [hunk 2]
```diff
@@ -143,24 +138,22 @@ more on the page. See below for a table. They override the symbols in [Survey To
...
 
 1. [CLDR-17694](https://unicode-org.atlassian.net/browse/CLDR-17694) - Back button in browser fails in forum under certain conditions
 2. [CLDR-17693](https://unicode-org.atlassian.net/browse/CLDR-17693) SurveyTool fatal in getDBConnection
 3. [CLDR-17658](https://unicode-org.atlassian.net/browse/CLDR-17658) - Dashboard slowness
-4. Images for the plain symbols. Non-emoji such as [€](https://cldr-smoke.unicode.org/smoketest/v#/fr/Symbols2/47925556fd2904b5), √, », ¹, §, ... do not have images in the info pane. \[[CLDR-13477](https://unicode-org.atlassian.net/browse/CLDR-13477)\]**Workaround**: Look at the Code column; unlike the new emoji, your browser should display them there.
+4. Images for the plain symbols. Non-emoji such as [€](https://cldr-smoke.unicode.org/smoketest/v#/fr/Symbols2/47925556fd2904b5), √, », ¹, §, ... do not have images in the Info Panell. \[[CLDR-13477](https://unicode-org.atlassian.net/browse/CLDR-13477)\]**Workaround**: Look at the Code column; unlike the new emoji, your browser should display them there.
```
Comment (macchiati) [inline docs/site/translation.md:148]: Panell => Panel

File: docs/site/translation.md [hunk 3]
```diff
@@ -169,41 +162,39 @@ This list will be updated as fixes are made available in Survey Tool Production.
...
-*Tip: The links in the [Info Panel](/translation/getting-started/guide#h.2jch1980f8sy) will point you to relevant instructions for the entry you're editing/vetting. Use it if in doubt.*
-
+    1. [Date & Time](translation/date-time/date-time-names)
+        - [Names](translation/date-time/date-time-names)
+        - [Patterns](translation/date-time)
+        - [Symbols](translation/date-time/date-time-symbols)
+    2. [Time zones](translation/time-zones-and-city-names)
+    3. [Plural forms](translation/getting-started/plurals)
+
+*Tip: The links in the [Info Panell](translation/getting-started/guide#info-panel) will point you to relevant instructions for the entry you're editing/vetting. Use it if in doubt.*
```
Comment (macchiati) [inline docs/site/translation.md:200]: Panell => Panel

File: docs/site/translation/getting-started.md [hunk 4]
```diff
@@ -6,19 +6,17 @@ title: Getting Started
...
-    - [Plurals & Units](/translation/getting-started/plurals)
-    - [Review Date & Time](/translation/date-time)
-    - [Review Numbers](/translation/number-currency-formats)
-    - [Review Zones](/translation/time-zones-and-city-names)
-    - [Data stability](/translation/getting-started/data-stability)
-    - [Empty cache](/translation/getting-started/empty-cache)
-
-\*If you (individual or your organization) have not established a connection with the CLDR technical committee, start with [Survey Tool Accounts](/index/survey-tool/survey-tool-accounts).
-
+    - [Survey Tool Guide](guide)
```
Comment (macchiati) [inline docs/site/translation/getting-started.md:11]: This doesn't work, as far as I can tell. 
It goes to 
https://github.com/unicode-org/cldr/blob/btangmu/t08038_b/docs/site/translation/guide
but it needs to go to 
https://github.com/unicode-org/cldr/blob/btangmu/t08038_b/docs/site/translation/getting-started/guide.md

That is, the .md files that correspond to a directory are at the same level as that directory, not inside it.

Same problem with various other links.

**Review by macchiati — CHANGES_REQUESTED** [review]

File: docs/site/translation/getting-started.md (hunk 4 above)
Comment (btangmu) [reply docs/site/translation/getting-started.md]: You're right. Changed to /translation/getting-started/guide

**Review by macchiati — APPROVED** [review]

**Pull request merged**